internal/services/costmanagement/anomaly_alert_resource.go

```diff
@@ -48,6 +48,13 @@ func (AnomalyAlertResource) Arguments() map[string]*pluginsdk.Schema {
 			ValidateFunc: commonids.ValidateSubscriptionID,
 		},
 
+		"email_address_sender": {
```

Comment:
could you explain what this property does as this is different from the API/SDK name here.
notification email and sender email seem rather different

my guess is it is the email the notification is sent from? if so perhapes it would be better as
```suggestion
		"notification_from_email_address": {
```

Reply:
Based on my understanding, this email is not the sender of the notifications. In the [Azure REST API specifications](https://github.com/Azure/azure-rest-api-specs/blob/0410d404c68289cb1737d06bba92133bb84b515c/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2023-08-01/scheduledActions.json#L760), the property is called `notificationEmail` and it is used to specify the email address of the person who should receive notifications related to the scheduled actions (e.g. if someone likes to unsubscribe from the notifications).

> Email address of the point of contact that should get the unsubscribe requests and notification emails.

I agree with you that it would be odd to have a different naming than [API](https://learn.microsoft.com/en-us/rest/api/cost-management/scheduled-actions/create-or-update-by-scope?view=rest-cost-management-2024-08-01&tabs=HTTP#request-body) / [SDK](https://github.com/hashicorp/go-azure-sdk/blob/b730ec95cadad74ec4ddb4f54482a2f66a4d9b2f/resource-manager/costmanagement/2023-08-01/scheduledactions/model_scheduledactionproperties.go#L10). I'll update the parameter to `notification_email` if that works for you.